includes/functions.php

```diff
@@ -1659,6 +1659,51 @@ function is_activity( $data ) {
 		)
 	);
 
+	if ( is_string( $data ) ) {
+		return in_array( $data, $types, true );
+	}
+
+	if ( is_array( $data ) && isset( $data['type'] ) ) {
+		return in_array( $data['type'], $types, true );
+	}
+
+	if ( is_object( $data ) && $data instanceof Base_Object ) {
+		return in_array( $data->get_type(), $types, true );
+	}
+
+	return false;
+}
+
+/**
+ * Check if an object is an Actor.
+ *
+ * @param array|object|string $data The object to check.
+ *
+ * @see https://www.w3.org/ns/activitystreams#actor
+ *
+ * @return boolean True if the object is an Activity, false otherwise.
```

Comment:
```suggestion
 * @see https://www.w3.org/ns/activitystreams#actor
 *
 * @param array|object|string $data The thing to check.
 *
 * @return boolean True if the passed argument is an Actor, false otherwise.
```